### ISSUE TITLE
Items not added using PreparedItemsForFeedbackManager delegate method

### ISSUE THREAD

**Issue by albertoms** [bitstadium/HockeySDK-Xamarin#50]
I'm defining a few items for the feedback message but they are not added to the message body.

This is my delegate class:

```
public class HockeyAppFeedbackDelegate : BITFeedbackManagerDelegate
    ...
        public override NSArray PreparedItemsForFeedbackManager (BITFeedbackManager feedbackManager)
        {
            NSArray items = new NSArray ();

            items.SetValuesForKeysWithDictionary (new NSDictionary(new NSString("hey"),new NSString ("please"),new NSString ("add"),new NSString ("me")));

            return items;
        }
    }
```

Then in another class I define this method:

```
public void ShowFeedbackUI ()
        {
            FeedbackManager = Manager.FeedbackManager;

            var feedbackController = FeedbackManager.FeedbackComposeViewController ();
            FeedbackDelegate = new HockeyAppFeedbackDelegate ();
            feedbackController.Delegate = FeedbackDelegate;

            var window = UIApplication.SharedApplication.KeyWindow;
            var viewController = window.RootViewController;

            while (viewController.PresentedViewController != null) {
                viewController = viewController.PresentedViewController;
            }

            UINavigationController navController = new UINavigationController (feedbackController);
            navController.ModalPresentationStyle = UIModalPresentationStyle.OverCurrentContext;
            viewController.PresentViewController (navController, true, null);
        }
```

But the items are not added to the message when I send the feedback.

Hope you can have any ideas on this :)

**Comment by TroubleMakerBen** [comment]
Hey @albertoms,

Thanks for getting in touch.
I'm really sorry for the incredible long silence on this. We just released 4.1.1 of the SDK and fixed the bindings for all the delegate methods for iOS. So please update to that. Once you upgraded, you need to do this:
Your implementation in `HockeyAppFeedbackDelegate` seems to be correct but don't assign the delegate to the FeedbackComposeViewController but on the umbrella class:

```csharp
var manager = BITHockeyManager.SharedHockeyManager;
manager.Configure(AppID);
var yourDelegate = new HockeyFeedbackDelegate();
manager.Delegate = yourDelegate;
manager.StartManager();
```

Let me know if you have additional questions and I promise that the response will be within an acceptable timeframe.

🖖

Benny

**Comment by TroubleMakerBen** [comment]
I'm closing this as we've fixed the bug in 4.1.1 of the SDK.

Thx for bringing this to my attention!

Benny
